src/target/target.h

```diff
@@ -117,8 +117,8 @@ struct gdb_service {
 
 /* target back off timer */
 struct backoff_timer {
-	int times;
-	int count;
+	int64_t next_attempt;
+	unsigned int interval;
```

Comment:
With next_attempt and the result from timeval_ms() being signed, and interval field being unsigned, I was wondering if there is the potential for signed/unsigned arithmetic sorts of issues with the calculation "bool poll_needed = timeval_ms() + polling_interval / 2 >= target->backoff.next_attempt;"    But I guess timeval_ms() is never going to return a value with a sign bit of 1, so no problem.

Reply:
Agreed that it's a bit odd, but timeval_ms() returns int64_t and that's why next_attempt is also signed.